client/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
let's have those 2 names changed to what the app is actually called :sweat_smile: